api-docs/client-interactions.md

```diff
@@ -0,0 +1,216 @@
+# Client Interactions
+
+The client interactions document direct encounters with the clients.
+
+Here is the list of the valid interaction types:
+
+- `clientInteractionCreated`,
+- `clientInteractionUpdated`,
+- `clientInteractionDeleted`,
+
+## Get a single Client Interaction
+
+### Request
+
+```http
+GET /api/clients/:id/interactions/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "title": "Nutrition Program - SNAP",
+  "serviceId": 7,
+  "interactionType": "walk-in",
+  "description": "Application successfully filled out. Decision pending.",
+  "date": "2019-05-06T05:00:00.000Z",
```

Comment:
If we're not tracking the time, let's not have a full timestamp here. See [birthday in clients](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/create-client.md) as an example. And see https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/backend/apis/clients/get-client.api.js#L132 for how we get the backend to only send the day.